backend/k8s-deployment.yml

```diff
@@ -0,0 +1,56 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: gosmartkickers-deploy
+spec:
+  selector:
+    matchLabels:
+      app: gosmartkickers
+  template:
+    metadata:
+      labels:
+        app: gosmartkickers
+    spec:
+      containers:
+        - name: gosmartkickers
+          image: smartkickers/backend:latest
+          ports:
+            - containerPort: 3000
+          imagePullPolicy: IfNotPresent
```

Comment:
With that configuration, if the tag `latest` gets updated with a new image, you will not be able to fetch it when redeploying on the same cluster. You have 2 options:
* You can switch to specific tags which you bump here, or
* You can change field `imagePullPolicy: Always`, so the image always gets pulled regardless if it's present or not

Reply:
The image name is prepared for the final step, but as we don ' have a cloud cluster and registry yet images are built by developers on their local machines it can't be pulled. "IfNotPresent" is a simple workaround for now.